src/NServiceBus.SagaAudit/SagaAuditFeature.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using System.Collections.Generic;
+    using NServiceBus;
+    using Pipeline;
+    using SagaAudit;
+
+    class SagaAuditFeature : Feature
+    {
+        public SagaAuditFeature()
+        {
+            DependsOn<Sagas>();
+            RegisterStartupTask<ServiceControlBackendInitializer>();
+        }
+        
+        protected override void Setup(FeatureConfigurationContext context)
+        {
+            context.Settings.TryGet("NServiceBus.SagaAudit.Serialization", out Func<object, Dictionary<string, string>> customSagaEntitySerialization);
```

Comment:
Side note: I LOVE this new syntax.

Actual comment: Could the `TryGet` ever return false? What happens if `customSagaEntitySerialization` is null?